src/VisualStudio/Core/Def/CallHierarchy/CallHierarchyItem.cs

```diff
@@ -151,8 +143,8 @@ private async Task NavigateToAsync()
         {
             using var context = _provider.ThreadOperationExecutor.BeginExecute(
                 ServicesVSResources.Call_Hierarchy, ServicesVSResources.Navigating, allowCancellation: true, showProgress: false);
-            await _provider.NavigateToAsync(
-                _symbolId, _workspace.CurrentSolution.GetProject(_projectId), context.UserCancellationToken).ConfigureAwait(false);
+            await _navigableLocation.NavigateToAsync(
+                NavigationOptions.Default with { PreferProvisionalTab = true }, context.UserCancellationToken).ConfigureAwait(false);
```

Comment:
what does preferprovisionaltab mean?

Reply:
it means that when you try to navigate to this it will pop open the provisional tab (on the RHS of the tab bar).  This tab will not stay open if you then navigate away from this.  Allowing easy navigation without clutter.  this matches the behavior of navigation here from before (you can see that in the deleted navigation method).

Reply:
Confusingly, its called the "Preview Tab" in the VS UI, but "Provisional Tab" in code 🤦‍♂️